polyfill/lib/civildate.mjs

```diff
@@ -25,13 +25,16 @@ export class CivilDate {
   with({ year = this.year, month = this.month , day = this.day } = {}) {
     return new CivilDate(year, month, day);
   }
-  withTime(time = CivilDateTime.now().toCivilTime()) {
+  withTime(time) {
     return new CivilDateTime.from(this, time);
   }
-  toString() {
+  asString() {
     const { year, month, day } = this;
     return `${spad(year, 4)}-${pad(month, 2)}-${pad(day, 2)}`;
   }
+  toString() {
+    return `[object CivilDate ${this.asString()}]`;
```

Comment:
This is a very strange result that I think would be a bad idea. If it's going to be in the `[object X]` form, it should follow that convention.

Rather than defining a toString, it would probably want `get [Symbol.toStringTag]() { return `CivilDate ${this.asString()`; }` - but I'd still think this is way less useful than returning the previous toString implementation.